environment.yaml

```
@@ -1,29 +1,32 @@
+  - python
   - requests
   - s3fs
+  - s5cmd
   - scipy
+  - tqdm
   - xarray
-  - zarr
-  - natsort
+  - zarr=2
+  - pip:
+    - river-route
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)